.github/workflows/snap.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       run: |
         sudo snap install --dangerous ${{ needs.build-snap.outputs.snap_name }}
         sudo snap connect fwupd:polkit :polkit
-        sudo fwupd.fwupdtool modify-config DisabledPlugins ""
+        sudo fwupd.fwupdtool enable-test-mode
```

Comment:
I like the concept, I just don't like the *name*. Mode implies one or the other, and this kinda just enables an extra optional part of fwupd.

Reply:
Got another preference?  Some ideas:
`*able-test-infrastructure`
`*able-test-devices`
`*able-virtual-devices`
`*able-virtual-test-devices`

Reply:
I guess the name needs to reflect what it does. It's really providing another couple of plugins that export new virtual devices we can play with. Maybe `enable-test-devices` is best?

Reply:
Or even `*able-testing` or `*able-test-suite`

Reply:
fixed